### ISSUE TITLE
Non-stop crashing in Android 6 in past 3 versions of Tachiyomi

### ISSUE THREAD

**Issue by buthub** [tachiyomiorg/tachiyomi#9401]
### Steps to reproduce

App often crashes (occasionally, it will have the prompt that the app has crashed, but most times not) every few minutes. The strange part is that it crashes on innocuous actions, and when I restart the app and do the same thing, it seems fine.

This seems to happen to my old Nexus 7 2013 tablet (Droid 6.0.1), but not my other more recent droid tablet. I can't stop using the Nexus 7 because it's the BEST tablet for reading manga!!! Please continue support for 6.0.1 devices!!

It crashes when changing categories, going to a manga title in the library, scrolling down the chapter list in a manga, clicking on a manga from search, going to webview, and many more situations. It seems like navigating around causes most of the crashing. Strangely, if I'm just reading the manga, it doesn't crash. 

### Expected behavior

It shouldn't crash.

### Actual behavior

It crashes like a mofo every few minutes of navigating around (not reading manga) for the most innocuous actions. So far, I've had hundreds and hundreds of crashes. (Yeah, I'm too poor/cheap to buy a new small android tablet with a good screen to replace my Nexus 7.)

### Crash logs

No option available to share/save the crash log in Android 6. The notification dropdown drawer only shows the crash log message, but doesn't have any way to save or share it.
EDIT: I tried getting the crash log directly, but it seems like it gets wiped every time it crashes, so the crash log is basically just the most basic info like the device name, etc. I tried getting the crash log several times, but it's always the same basic info. No info about what caused the crash itself.

### Tachiyomi version

0.14.6 (and also previous 2+? versions)

### Android version

Android 6.0.1

### Device

Nexus 7 2013

### Other details

I know it's an old tablet, but with its gorgeous screen and small form, it's the perfect manga reader! Please continue support for it. 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have gone through the [FAQ](https://tachiyomi.org/help/faq/) and [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.14.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by LordFrz** [comment]
Same issue, same device. Issue was was not happening with the beta preview version. I thought the latest update would fix it, but it did not. Worked perfectly for year, teach me to update apps.

**Comment by Seishirou101** [comment]
Please provide crash logs if possible.

Potential ways of getting them:

- More → Settings → Advanced → Dump crash logs
- https://help.getpocket.com/article/1073-retrieving-a-pocket-crash-log-on-android
- https://play.google.com/store/apps/details?id=com.dp.logcatapp&hl=en_CA
    
for the first option, you could possibly grab those logs from `Android/data/eu.kanade.tachiyomi/cache` I believe if you can't share them normally

**Comment by amayra** [comment]
i have same problem but i forget to report back

**Comment by LordFrz** [comment]
[tachiyomi_crash_logs.txt](https://github.com/tachiyomiorg/tachiyomi/files/11471881/tachiyomi_crash_logs.txt)

Heres my crash log.

**Comment by LordFrz** [comment]
I was able to read 40 chapters of an old manga I downloaded a year ago without crashing up till I was caught up, recent chapters chain crashed. So it could be with how new chapters download. When I switched off download as cbz archive I did not crash anymore on the new chapters. I'll test that for a few days and see if it's a coincidence or the cause. But give that a try too.

TLDR: try turning off download as cbz archive in setting/downloads

**Comment by arkon** [comment]
You could try seeing if the preview builds are any better.

**Comment by arkon** [comment]
The only idea I have here is that the app's become too heavy for weaker devices, probably due to the usage of Jetpack Compose. We've already been updating Compose as new releases come out to take advantage of its optimizations, but I'm not really willing to trade the improved developer experience to potentially support some older devices. :\

Also FYI we're dropping Android 6-8 for a future v0.16.0+ since that's what the Android landscape is already shifting towards more broadly.

**Comment by LordFrz** [comment]
> Also FYI we're dropping Android 6-8 for a future v0.16.0+ since that's what the Android landscape is already shifting towards more broadly.

good to know, ill start looking for an alternative and backup my libraries to somthing more long term.
